Sparkle.xcodeproj/project.pbxproj

```diff
@@ -2821,8 +2825,11 @@
 				CLANG_ENABLE_MODULES = YES;
 				COMBINE_HIDPI_IMAGES = YES;
 				DEFINES_MODULE = YES;
-				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks @loader_path/Frameworks";
-				MACOSX_DEPLOYMENT_TARGET = 10.10;
+				LD_RUNPATH_SEARCH_PATHS = (
```

Comment:
I think a bunch of this is Xcode updating the _format_ of settings while it's at it. It's now or some other time.